getting_started/hello_world.md

```diff
@@ -31,7 +31,7 @@ echo "Hello World!"
 
 1) `#!/bin/bash` This is a [shebang] it tells Linux how to handle the file.
 Here's we're saying use `bash` to run this file
-2) `#SBATCH --cpus=1` This is a SLURM directive asking for 1 CPU
+2) `#SBATCH -n=1` This is a SLURM directive asking for 1 CPU
```

Comment:
`-n` or `--ntasks` sets the maximum number of task that the batch job will launch
`--cpus` no longer looks to be a flag

Maybe `--ntasks=1` and `--cpus-per-task=1` to keep it similar to the original intent

Reply:
from slurm docs:

```-c, --cpus-per-task=<ncpus>
Advise the Slurm controller that ensuing job steps will require ncpus number of processors per task. Without this option, the controller will just try to allocate one processor per task.```

We only have 2 options in this bash script. We're already assuming default time, account, partition, no tmp space, etc. So at some point we have to decide what defaults we want to use and those we do not. By specifying `-c`, we're simply overriding a default option with a default. If you want to change something here, just add `-c 1` but again, its just making explicit a default.

Now, I didn't notice that the syntax is still wrong here @tinosulzer when I approved this. In slurm, two dashes needs an equals and one doesn't. So pls change that.